components/molecule/dataCounter/src/index.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import AtomButton from '@schibstedspain/sui-atom-button'
+import AtomInput, {inputSizes} from '@s-ui/react-atom-input'
+import MoleculeField from '@s-ui/react-molecule-field'
+
+import {withStateValue} from '@s-ui/hoc'
+
+const BASE_CLASS = `sui-MoleculeDataCounter`
+const CLASS_INPUT_CONTAINER = `${BASE_CLASS}-container`
+
+const MoleculeDataCounter = ({
+  id,
+  label,
+  value,
+  errorText,
+  size = inputSizes.MEDIUM,
+  charsSize = 2,
+  placeholder = '1',
+  max = 99,
+  min = 1,
+  minValueHelpText = 'Minimum Value',
+  maxValueHelpText = 'Maximum Value',
+  onChange,
+  disabled
+}) => {
+  value = value || min
+
+  const isBelowMaxValue = value < max
+  const isOverMinValue = value > min
+  const isMaxValue = value === max
+  const isMinValue = value === min
+
+  const incrementValue = e => {
+    if (isBelowMaxValue) onChange(e, {value: value + 1})
+  }
+
+  const decrementValue = e => {
+    if (isOverMinValue) onChange(e, {value: value - 1})
+  }
+
+  const handleKeyDown = e => {
```

Comment:
Copied from Paper: "Input field value can also be changed by the user using the keyboard so Input field must receive focus as well as the buttons, but only number values are allowed."

Should numbers keys have to be allowed? I think that should be

Reply:
Done!